src/lib/php/common-sysconfig.php

```diff
@@ -602,6 +602,12 @@ function Populate_sysconfig()
   $valueArray[$variable] = array("'$variable'", "'/licenses/export'", "'$prompt'",
     strval(CONFIG_TYPE_TEXT), "'LicenseDB'", "3", "'$desc'", "null", "null");
 
+  $variable = "LicenseDBSleep";
+  $prompt = _('Maximum sleep time');
+  $desc = _('Maximum time to wait in seconds');
+  $valueArray[$variable] = array("'$variable'", "'10'", "'$prompt'",
+    strval(CONFIG_TYPE_INT), "'LicenseDB'", "4", "'$desc'", "null", "null");
+
```

Comment:
This is not required for this, Needs to be removed from all the places. More of a unused variable.

Reply:
Remove Line 58 as well :)

Reply:
Sure will do that.